MetaMask/metamask-extension
# Update controllers with conversionRate change with minimal required changes in extension

Fixes: #9492

Explanation:  My original [PR to fix this issue](https://github.com/MetaMask/metamask-extension/pull/11066) got, as @brad-decker gently pointed out to me, a bit out of hand. This PR aims to add the core functionality needed to land [the changes in the controller's repo](https://github.com/MetaMask/controllers/pull/465) for the extension - which set conversionRate to null when we don't have a valid conversion rate from the current network's native currency to the user's preferred currency (usually fiat).
 
Manual testing steps:
- In advanced settings, turn `Show Conversion on Testnet`s on.
- Go between mainnet, a custom rpc network where you have added a ticker symbol and a custom rpc network where you haven't added a ticker symbol.
- Check that conversions are correct for all assets across these different networks (for the network with no ticker symbol there should be no conversions).
